primevue-gen/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "primvue-gen",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Shouldn't we use the same license that we use for Promgen? (MIT)
In fact, is this field actually necessary? If it can be removed, I would prefer that, because we already have a `LICENSE` file at the root of the repo.

Reply:
I see you have addressed this commit with 43381cccbaa8ee644d3c72c043c22c552fec6586.

This conversation is now resolved.